config/deploy.rb

```diff
@@ -2,7 +2,7 @@
 lock '3.3.5'
 
 set :application, 'cakeside'
-set :repo_url, 'git@github.com:mokhan/cakeside.git'
+set :repo_url, 'git@github.com:cakeside/cakeside.git'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.